mitodl/ocw-hugo-themes
# remove course-v3 and course-offline-v2

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/ocw-hugo-themes/issues/1013

#### What's this PR do?
In https://github.com/mitodl/ocw-hugo-themes/pull/1068, we added `course-v3` and `course-offline-v2`, which were barebones copies of `course-v2` and `course-offline` respectively. This was done to support research into unifying the online and offline builds into one build that generates relative URLs that can be used anywhere. We've decided to go in a different direction and keep the online and offline builds separate, so this PR removes `course-v3` and `course-offline-v2`.

#### How should this be manually tested?
This shouldn't require much testing as the new themes are just skeleton bare bones copies of their progenitors. They're not being used anywhere. Run `yarn start course` and `yarn start www` and ensure that a course site and `ocw-www` start properly.
